From 0f3760eeb30ee783c6711d586469409265d13ff3 Mon Sep 17 00:00:00 2001
From: Aleksandrs Vinarskis <alex.vinarskis@gmail.com>
Date: Sat, 29 Mar 2025 13:06:24 +0100
Subject: [PATCH 08/12] usb: typec: Add Parade PS8833 Type-C Retimer variant

Appears to behave similarly to Parade PS8830. Found on some Qualcomm
Snapdragon X1 devices, such as Asus Zenbook A14.

Signed-off-by: Aleksandrs Vinarskis <alex.vinarskis@gmail.com>
---
 drivers/usb/typec/mux/ps883x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/typec/mux/ps883x.c b/drivers/usb/typec/mux/ps883x.c
index ad59babf7cce..095c36530904 100644
--- a/drivers/usb/typec/mux/ps883x.c
+++ b/drivers/usb/typec/mux/ps883x.c
@@ -447,6 +447,7 @@ static void ps883x_retimer_remove(struct i2c_client *client)
 
 static const struct of_device_id ps883x_retimer_of_table[] = {
 	{ .compatible = "parade,ps8830" },
+	{ .compatible = "parade,ps8833" },
 	{ }
 };
 MODULE_DEVICE_TABLE(of, ps883x_retimer_of_table);
-- 
2.45.2

